### PR TITLE
Desktop view

### PR DESCRIPTION
**Pull request by sireclifford**
Added animations to buttons

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,9 @@
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
+https://sireclifford.github.io/sire-portfolio/
+
```
Comment (elyor-doniyorov) [inline README.md]: - [x] Kindly create `Live Demo` section above `Getting Started` section and copy the live link from line 19 and paste it in `Live Demo` section. Also don't forget to change the link inside `Getting Started` section with GitHub repo link.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @sireclifford, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ There are no linter errors
✔️ The website is deployed using GitHub pages
✔️ The page works fine

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- to update PR title and description to adapt to current milestone which is deployment.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sireclifford) [reply README.md]: Thank you @elyor-doniyorov. I have implemented the changes

File: README.md [hunk 2]
```diff
@@ -11,12 +11,20 @@
 - CSS
 - Node.Js
 
+## Live Demo (if available)
 
+[Live Demo Link](https://livedemo.com)
```
Comment (mikethreels) [inline README.md]: - [ ] this link should be the actual link to your live demo, not the placeholder link.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Sire, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @sireclifford, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ There are no linter errors
✔️ The live link works fine
✔️ etc.

## Optional suggestions
- [OPTIONAL] Please update your PR title and description to adapt current milestone which is deployment. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**